contracts/abstract/Version.sol

```diff
@@ -0,0 +1,6 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
```

Comment:
spotted there are other abstract contracts in respective directory - would this be better in a base or version directory

Reply:
> spotted there are other abstract contracts in respective directory - would this be better in a base or version directory

It depends where else they are used. Currently they are only used by contracts under `bond`. If they are ever used by contract other than those under `bond` they can be promoted to `common` area.